x-pack/plugin/inference/src/test/java/org/elasticsearch/xpack/inference/services/AbstractInferenceServiceParameterizedTests.java

```diff
@@ -0,0 +1,387 @@
+/*
```

Comment:
The reason this class exists and isn't included in `AbstractInferenceServiceTests` is adding parameterized tests in `AbstractInferenceServiceTests` results in the the non parameterized tests being run for each permutation, even though those tests don't depend on the parameterized test information.

It seemed cleaner to separate them 🤷‍♂️